notebooks/notebooks/iclr2022/winrate-predictions.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        """Return (<code>win_rates</code>, <code>gt_wins</code>) as a tuple."""
</pre><p>what does <code>gt_wins</code> stand for? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
fixed